File: Hahn.ApplicatonProcess.February2021.Domain/Models/AssetModel.cs
using System;
using Hahn.ApplicatonProcess.February2021.Data.Enums;

namespace Hahn.ApplicatonProcess.February2021.Domain.Models
{
    public class AssetModel
    {
        public int ID { get; set; }
        public string AssetName { get; set; }
        public DepartmentEnum Department { get; set; }
        public string CountryOfDepartment { get; set; }
        public string EMailAdressOfDepartment { get; set; }
        public DateTime PurchaseDate { get; set; }
        public bool Broken { get; set; }
    }
}

File: Hahn.ApplicatonProcess.February2021.Web/ClientApp/src/models/asset.model.ts
export class AssetModel {
  id: number;
  assetName: string;
  department: DepartmentEnum;
  countryOfDepartment: string;
  eMailAdressOfDepartment: string;
  purchaseDate: Date;
  broken: boolean;
}

export enum DepartmentEnum {
  HQ = 1,
  Store1,
  Store2,
  Store3,
  MantenanceStation,
}
